### PR TITLE
Bug 1798646: Backport fix for incorrect channel displayed in OperatorHub

### PR DESCRIPTION
**Pull request by TheRealJon**
Backport #3536 fix for https://bugzilla.redhat.com/show_bug.cgi?id=1764540

### CODE REVIEW

File: frontend/public/components/operator-lifecycle-manager/index.tsx [hunk 1]
```diff
@@ -218,6 +218,7 @@ export type PackageManifestKind = {
       name: string;
       currentCSV: string;
       currentCSVDesc: {
+        description?: string;
```
Comment (josephschorr) [inline frontend/public/components/operator-lifecycle-manager/index.tsx]: Why this change?

Comment (spadgett) [reply frontend/public/components/operator-lifecycle-manager/index.tsx]: FWIW, `oc explain packagemanifests.status.channels.currentCSVDesc` shows all of these fields as optional. This might be a mistake in the schema, though.

Comment (TheRealJon) [reply frontend/public/components/operator-lifecycle-manager/index.tsx]: The missing description property was causing a typescript error.

**Comment by TheRealJon** [comment]
/retest

Test flake:
```sh
1) Kubernetes resource CRUD operations Route : edit the resource instance
   Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
       at ontimeout timers.js:436:11
       at tryOnTimeout timers.js:300:5
       at listOnTimeout timers.js:263:5
       at Timer.processTimers timers.js:223:10
```
